### ISSUE TITLE
Is it possible to inject a set of Markdown files not from Notion?

### ISSUE THREAD

**Issue by meefen** [hwchase17/notion-qa#4]
If so, what preprocessing is needed?

**Comment by nance2uiuc** [comment]
I would also like to know.

**Comment by jasielmacedo** [comment]
Yes, you can. The Notion DB folder has a lot of *.md files, which are basically a lot of markdown files, so you can make your own.

You don't have to change the code; 
Remember that the Open AI embedding can read any kind of text.
just make sure that every file you make has a proper title

**Comment by meefen** [comment]
@jasielmacedo Thanks a lot for your response. 

Can you say more about:
> every file you make has a proper title

Are we talking about any Markdown header info or simply having a level 1 heading?

**Comment by jasielmacedo** [comment]
Yeah, to Simply having Level 1 heading in it will be enough
